rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -2065,7 +2065,7 @@ void opcode_00ae(const ScriptArguments& args, const ScriptVehicle vehicle, const
     {
```

Comment:
I'd consider adding a ``VehicleObject::getDriver()`` that returns a ``CharacterObject*``. As there's also code in the obstacle avoiding method that performs similar checks, this would tidy up some of the casting so it's a bit easier to follow what's going on.